src/main/java/com/hedera/hashgraph/sdk/TransactionReceipt.java

```diff
@@ -50,6 +51,14 @@ public ContractId getContractId() {
         return new ContractId(inner.getContractIDOrBuilder());
     }
 
+    public TopicId getTopicId() {
```

Comment:
Add this as a final variable, the other getters should have been deprecated in a recent PR iirc